requests/utils.py

```diff
@@ -665,6 +665,15 @@ def parse_header_links(value):
 
     return links
 
+def is_valid_location(response):
+    """Verify that multiple Location headers weren't
+    returned from the last response.
+    """
+    if hasattr(response.raw, 'headers'):
+        if hasattr(response.raw.headers, 'getlist'):
```

Comment:
Yeah, I'm onboard with this. I updated the PR with a somewhat verbose implementation of `getattr`.